nemo/collections/nlp/losses/cross_entropy.py

```diff
@@ -14,89 +14,62 @@
 # limitations under the License.
 # =============================================================================
 
+from typing import List, Optional
+
 import torch
 from torch import nn
 
-from nemo.core.classes import NeuralModule, Typing, typecheck
+from nemo.core.classes import Serialization, Typing, typecheck
 from nemo.core.neural_types import LabelsType, LengthsType, LogitsType, LogprobsType, LossType, MaskType, NeuralType
 from nemo.utils.decorators import experimental
 
-__all__ = ['SmoothedCrossEntropyLoss', 'CrossEntropyLoss']
-
+__all__ = ['SmoothedCrossEntropyLoss']
 
-def mask_padded_tokens(tokens, pad_id):
-    mask = tokens != pad_id
-    return mask
 
-
-# @experimental
-class CrossEntropyLoss(NeuralModule):
+@experimental
+class SmoothedCrossEntropy(torch.nn.Module):
```

Comment:
rename the file with SmoothedCrossEntropy accordingly? Should it inherit from nemo.core.Loss? If so, can we combine 2 classes?

Reply:
i will inherit SmoothedCrossEntropyLoss from nemo.core.Loss, SmoothedCrossEntropy is just a pytorch nn.module